### PR TITLE
Added Hello Evie button. Still working through tutorial for the array

### PR DESCRIPTION
**Pull request by egrimshaw**
I added a function and a button to the main page to display Hello Evie to show the servlet has been connected. I had to replace the h1 tags with spaces - unsure if there is a better way of doing it? 

I am still working through a step of the walkthrough, which is why I have a hard coded array in the servlet file right now. 

Also, is it possible to setContentType twice? I would like the Hello Evie to displayed as HTML and then the array as a JSON string, but once I re set the content type the Hello Evie tag changes.

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -28,5 +31,13 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
     response.getWriter().println("<h1>Hello Evie!</h1>");
```
Comment (arjun-garg-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:33]: This line looks like why you need to strip the h1 headers from the fetched text

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -28,5 +31,13 @@
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     response.setContentType("text/html;");
```
Comment (arjun-garg-google) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:32]: you don't need to set the content type to HTML if you are planning to return json. there is no way to set the content type twice. My recommendation is to just return all the data as json and then perform an html formatting/rendering you need post-fetch.

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -29,9 +29,9 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello Evie!</h1>");
+   // document.write("<html><h1>Hello Evie!</h1></html>");
```
Comment (gwroblew) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:32]: Not needed?

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -44,8 +44,26 @@ function addNavBar(){
...
-    response.replace("<h1>", ' '). replace("</h1>", ' ')).then((greeting) =>
-    {document.getElementById('hello-container').innerText = greeting; 
+    fetch('/data').then(response => response.json()).then((arrayString) =>
+    {
+        const helloElement = document.getElementById('hello-container');
+        helloElement.innerHTML = '';
+        helloElement.appendChild(createHeading(arrayString[0]));
+        helloElement.appendChild(createList('First Comment: ' + arrayString[1]));
+        helloElement.appendChild(createList('Second Comment: ' + arrayString[2]));
+        helloElement.appendChild(createList('Third Comment: ' + arrayString[3]));
```
Comment (gwroblew) [inline portfolio/src/main/webapp/script.js:54]: Should this be a loop?

Comment (egrimshaw) [reply portfolio/src/main/webapp/script.js:54]: Hi! I will make this change for the future functions I do - thank you!

**Pull request merged**